2023-04-26 13:50:05 [1m[31mERROR[0m HTTPSConnectionPool(host='nohminatim.openstreetmap.org', port=443): Max retries exceeded with url: /search?q=valapolku%2B9&format=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000026AA58C7550>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))